[2024-01-26T18:47:45.346+0530] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: sample_dag_v3.2nd_task scheduled__2024-01-16T00:00:00+00:00 [queued]>
[2024-01-26T18:47:45.353+0530] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: sample_dag_v3.2nd_task scheduled__2024-01-16T00:00:00+00:00 [queued]>
[2024-01-26T18:47:45.354+0530] {taskinstance.py:2170} INFO - Starting attempt 1 of 6
[2024-01-26T18:47:45.383+0530] {taskinstance.py:2191} INFO - Executing <Task(BashOperator): 2nd_task> on 2024-01-16 00:00:00+00:00
[2024-01-26T18:47:45.386+0530] {standard_task_runner.py:60} INFO - Started process 6702 to run task
[2024-01-26T18:47:45.390+0530] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'sample_dag_v3', '2nd_task', 'scheduled__2024-01-16T00:00:00+00:00', '--job-id', '113', '--raw', '--subdir', 'DAGS_FOLDER/1-sample-dag.py', '--cfg-path', '/tmp/tmptcekdsdp']
[2024-01-26T18:47:45.391+0530] {standard_task_runner.py:88} INFO - Job 113: Subtask 2nd_task
[2024-01-26T18:47:45.447+0530] {task_command.py:423} INFO - Running <TaskInstance: sample_dag_v3.2nd_task scheduled__2024-01-16T00:00:00+00:00 [running]> on host mh1.myguest.virtualbox.org
[2024-01-26T18:47:45.545+0530] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='hdu' AIRFLOW_CTX_DAG_ID='sample_dag_v3' AIRFLOW_CTX_TASK_ID='2nd_task' AIRFLOW_CTX_EXECUTION_DATE='2024-01-16T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-01-16T00:00:00+00:00'
[2024-01-26T18:47:45.546+0530] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-01-26T18:47:45.548+0530] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo I am second task and will run after first task ']
[2024-01-26T18:47:45.557+0530] {subprocess.py:86} INFO - Output:
[2024-01-26T18:47:45.557+0530] {subprocess.py:93} INFO - I am second task and will run after first task
[2024-01-26T18:47:45.558+0530] {subprocess.py:97} INFO - Command exited with return code 0
[2024-01-26T18:47:45.582+0530] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=sample_dag_v3, task_id=2nd_task, execution_date=20240116T000000, start_date=20240126T131745, end_date=20240126T131745
[2024-01-26T18:47:45.646+0530] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-01-26T18:47:45.664+0530] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
